### ISSUE TITLE
Add considerations to Media Push client-side docs

### ISSUE THREAD

**Issue by kelzr** [AgoraIO/Docs-Source#1445]
Add considerations to Media Push client-side docs (https://docs.agora.io/en/3.x/interactive-live-streaming/advanced-features/media-push?platform=android): A streaming task refers to a single streaming operation. The maximum concurrent task limit (PCW) is set to 50 by default, which means you can have up to 50 streaming tasks running simultaneously within a single Agora project. If you require a higher quota,  reach out to our technical support.

This is a request from R&D.

**Comment by saudsami** [comment]
Hi @kelzr 
Do you want this text added to the 3.x docs?
Please confirm.

**Comment by kelzr** [comment]
> Hi @kelzr
> Do you want this text added to the 3.x docs?
> Please confirm.

Yes, this information is helpful for the media push feature. Therefore, please include it in both the media push doc and the 3.x doc, as the latter is currently undergoing maintenance.

**Comment by saudsami** [comment]
The 4.x media push doc contains the following text:
**Maximum number of concurrent tasks**
The default maximum number of concurrent tasks is 20, which indicates that a maximum of 20 Media Push tasks can be run simultaneously in one project. If a higher quota is required, contact technical support.
https://docs.agora.io/en/media-push/develop/integration-best-practices#maximum-number-of-concurrent-tasks
Should I change 20 to 50?

**Comment by kelzr** [comment]
My issue refers to the client-side PCW restrictions, but the link you provided (https://docs.agora.io/en/media-push/develop/integration-best-practices#maximum-number-of-concurrent-tasks) is for the server-side RESTful PCW restrictions. It seems like you've confused the concepts.

**Comment by saudsami** [comment]
Here's the staging link to the updated doc. 
https://docs-staging-git-1445-add-considerations-to-m-239432-agora-gdxe.vercel.app/en/3.x/interactive-live-streaming/advanced-features/media-push?platform=android#considerations
Please let me know if you need anything further.

**Comment by kelzr** [comment]
LGTM
